src/pages/api/auth/[...nextauth].ts

```diff
@@ -43,6 +43,7 @@ export default NextAuth({
 				...session,
 				user: { ...session.user, nickname: token.nickname },
 				id: token.sub,
+				access_token: token.cloud_idp_access_token,
```

Comment:
Had to change auth here to expose the `access_token` to the frontend.

We need to pass this entire token so it can be validated on the API side to ensure it was signed by our trusted IDP.